src/App.js

```diff
@@ -13,7 +12,9 @@ import Contact from 'screens/Contact/Contact';
 import Terms from 'screens/Terms/Terms';
 import Privacy from 'screens/Terms/Privacy';
 import Embed from 'screens/Embed/Embed';
-import CompareModels from 'screens/CompareModels/CompareModels';
+import AllStates from 'screens/internal/AllStates/AllStates';
+import CompareModels from 'screens/internal/CompareModels/CompareModels';
+import ShareImage from 'screens/internal/ShareImage/ShareImage';
```

Comment:
I reorganized all the 'internal' pages to be under an `internal/` folder and changed the routing to match (see below) but left redirects in place so the old URLs work too.